app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -24,8 +35,26 @@ const DelegationValidation = ({delegationValidationError, txSuccessTab}) =>
     )
   )
 
-const StakePoolInfo = ({pool, gettingPoolInfo}) => {
-  const {fixedCost, homepage, margin, ticker, name, validationError} = pool
+type StakePoolLabelProps = {
+  isTicker: boolean
+  isPoolHash: boolean
+}
+
+const StakePoolLabel = ({isTicker, isPoolHash}: StakePoolLabelProps): h.JSX.Element => (
+  <Fragment>
+    <span className={isTicker ? 'valid' : ''}>Ticker</span> or{' '}
+    <span className={isPoolHash ? 'valid' : ''}>Stake Pool ID</span>
+  </Fragment>
+)
+
+type StakePoolInfoProps = {
+  pool?: Stakepool
+  gettingPoolInfo: boolean
+}
+
+const StakePoolInfo = ({pool, gettingPoolInfo}: StakePoolInfoProps): h.JSX.Element => {
+  const {fixedCost, homepage, margin, ticker, name, poolHash, validationError} =
+    pool === null ? ({} as Stakepool) : pool
```

Comment:
If we reset the `selectedPool` to dummy empty string we dont have to check here if its null.
Notice in all these places we do something like 
```
${ticker || ''}
```
and we also would't have to do that.

Reply:
See comment above. Although I'm not really happy with this line of code.

Reply:
We could create a dummy `Stakepool` object with all the fields empty, and that way we could remove all the `|| '' ` and also the `as Stakepool` since it would be a valid stakepool.

Reply:
Refactored